packages/ui/src/core/Image.tsx

```diff
@@ -64,7 +64,7 @@ export interface ImageProps extends Omit<NextImageProps, 'loading'> {
 const Image: React.FC<ImageProps> = (props) => {
   const {
     loading: injectedLoading,
-    ar,
+    ar = "16:9",
```

Comment:
`ar` can not be falsy value, because we need the css:

https://github.com/gravis-os/gravis-os/blob/e6903d251285c00f523e8a18712425c476323f40/packages/ui/src/core/Image.tsx#L206-L214